docs/clipboard-unsanitized/explainer.md

```diff
@@ -214,32 +151,17 @@ const blobOutput = await clipboardItems[0].getType('text/html');
 
 ```
 
-### Clipboard output on Windows
-
-```
-Version:0.9
-StartHTML:0000000105
-EndHTML:0000000436
-StartFragment:0000000400
-EndFragment:0000000400
-<!--StartFragment-->
-<html>
-<head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"><meta name="ProgId" content="Excel.Sheet"><meta name="Generator" content="Microsoft Excel 15"><style>body {font-family: HK Grotesk; background-color: var(--color-bg);}</style>
-</head>
-<body><div>hello</div>
-</body>
-</html>
-<!--EndFragment-->
-
-```
-
 ## Privacy and Security
 This feature introduces an `unsanitized` option that has unsanitized `text/html` content. This will be exposed to both native apps and websites.
 
-Websites or native apps are already reading unsanitized content via DataTransfer APIs using `setData()` & `getData()` methods. In this proposal, web authors are required to explicitly specify `unsanitized` option in the async clipboard `read()` method to access the raw `text/html` content from the clipboard. This feature uses async clipboard API that already has a [user gesture requirement](https://w3c.github.io/clipboard-apis/#check-clipboard-read-permission) on top of [existing](https://github.com/dway123/clipboard-pickling/blob/main/explainer.md#permissions) async clipboard API security measures to mitigate security and privacy concerns.
+Websites or native apps are already reading unsanitized content via DataTransfer APIs using `getData()` method. In this proposal, web authors are required to explicitly specify `unsanitized` option in the async clipboard `read()` method to access the raw `text/html` content from the clipboard. This feature uses async clipboard API that has a [user gesture requirement](https://w3c.github.io/clipboard-apis/#check-clipboard-read-permission) on top of [existing](https://github.com/dway123/clipboard-pickling/blob/main/explainer.md#permissions) async clipboard API security measures to mitigate security and privacy concerns.
 
 For more details see the [security-privacy](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/ClipboardAPI/tag-security-privacy-clipboard-unsanitized-read.md) doc.
 
+[Here](https://docs.google.com/document/d/1QLt50Q8UnlQksVltZ2PNkDZVdk9N58Pq7P0lzGTKh-U/edit?usp=sharing) is a threat model document for this feature.
```

Comment:
This document was made for Chromium T&S reviews. I don't think we should include a link in the explainer without sanitization (pun intended :) ). I don't think it is strictly necessary here given that you have described the security considerations elsewhere. So I would actually suggest just leaving it out.